src/components/images/imageLoader.js

```diff
@@ -45,13 +127,8 @@ import 'css!./style';
                 elem.setAttribute('src', url);
             }
 
-            if (userSettings.enableFastFadein()) {
-                elem.classList.add('lazy-image-fadein-fast');
-            } else {
-                elem.classList.add('lazy-image-fadein');
-            }
```

Comment:
This breaks the animation if no blurhash exists or if the user disables them (which is planned).